src/ControlSystem/ControlErrors/Rotation.hpp

```diff
@@ -12,6 +12,7 @@
 #include "ControlSystem/Tags.hpp"
 #include "DataStructures/DataBox/DataBox.hpp"
```

Comment:
[random line]

While you're fixing the build error, could you shorten the commit message so it's under 72 characters? Maybe just
```
Change return type of ExcisionSphere::center()
```